docs/_posts/2016-08-19-fund-10-Oth-prereq.md

```diff
@@ -1,12 +1,153 @@
+
+Wait until the deployment complete. 
+
+
+**3.** Obtain the public IP of the Jenkins master VM. 
+
+On the Azure portal, click on **Resource Group** and look for the Resource Group that you have just created.
+
+Click on the virtual machine in the resource group (_pumrp-jenkins_ in this example) and look for the "Public IP address/DNS name label".
+
+![Obtain the public IP of the Jenkins Master](<../assets/jenkins/jenkinsmaster_ip.png>)
+
+**4.** SSH to the Jenkins Master VM
+
+Open PuTTY (or any other ssh tool that you like) and type the following information to logon to the VM:
+```
+jenkinsadmin@fqdn_of_your_jenkinsmaster
+In from the screen capture above, it would be: jenkinsadmin@pumrp-jenkins.westus.cloudapp.azure.com 
+```
+
+To make this lab easier, the user has been pre-configued to be **jenkinsadmin** through the automated deployment. 
+
+![SSH to the Jenkins Master VM](<../assets/jenkins/putty_to_jenkinsmaster.png>){:height="350px"}
+
+Then type the password that you have specified.
+
+### Task 2: Configure your Jenkins Master
+In this task, we will perform the basic configuration of the Jenkins master server and install the necesary plugins that will be used for Continuous Integration.
+
+**1.** Obtain the initial admin password
+
+From the SSH session that you have opened at the end of Task 1, type the following command to obtain the initial admin password.
+
+``` bash
+sudo cat /var/lib/jenkins/secrets/initialAdminPassword
+```
+
+Copy the value returned by the command.
+Keep the SSH session open, we will return at the end of this task.
+
+![Initial jenkins admin password](<../assets/jenkins/initial_jenkins_password.png>) 
+
+**2.** Unlock the jenkins master
+
+With your browser, navigate to the default page of the Jenkins master. 
+
+```
+http://fqdn_of_your_jenkinsmaster:8080
+```
+
+Paste the initial admin password obtained earlier to unlock your instance of Jenkins and click **Continue**.
+
+![Initial jenkins admin password](<../assets/jenkins/initial_jenkins_unlock.png>){:width="500px"}
+
+Click **Install suggested plugins** 
+
+![Initial jenkins plugins](<../assets/jenkins/initial_jenkins_plugins.png>){:width="500px"}
+
+**3.** Create the first user 
+
+Create a user from the "Create First Admin User": 
+* Username = jenkins
+* Password = Passw0rd
+* Click **Save and Finish**. 
+
+![Initial jenkins plugins](<../assets/jenkins/first_jenkins_user.png>){:width="500px"}
+
+**4.** Restart Jenkins
+
+Click the **Restart** button to restart Jenkins. 
+
+![Initial jenkins plugins](<../assets/jenkins/restart_jenkins.png>){:width="500px"}
+
+You now have a virtual machine in Azure running Jenkins as a Master.
+
+**5.** Logon on the jenkins master with the credentials that you have just created in step 4. 
+
+**6.** Navigate to the "Configure System" page:
+
+```
+http://fqdn_of_your_jenkinsmaster:8080/configure
+```
+Look for the Jenkins URL field and type the URL of your Jenkins master: http://fqdn_of_your_jenkinsmaster:8080/
+
```

Comment:
ohhhh... Indeed, now my CD works... it's weird...
Maybe, we can insist on the fact that we have to re-enter it to avoid stupid mistake like I did... 🦆 lol